### ISSUE TITLE
v0.4.0 First impressions

### ISSUE THREAD

**Issue by Widdershin** [ticktackim/ticktack-workplan#160]
Hiya!

Congratulations on your v0.4.0 beta release.

Thought I would download it and have a poke around.

Here are some relatively raw first impressions for you:

After opening Tick Tack, I was greeted with the new user page. I was quite nervous about this page after my previous adventure of accidentally forking my feed with Patchwork, so I cancelled out of it.

I was then keen to poke around and see what content in tick tack looks like.

Unfortunately my discover feed is empty!

<img width="1440" alt="screen shot 2018-06-04 at 10 02 54 am" src="https://user-images.githubusercontent.com/398365/40911441-0bedf610-67ef-11e8-823c-beaaea77831d.png">

It's not clear to me if syncing is in progress, or if anything will ever appear here, or if it's because I haven't done something?

I figured then that I should subscribe to some stuff, so I click the Subscriptions button. I see this:

<img width="1440" alt="screen shot 2018-06-04 at 12 00 08 pm" src="https://user-images.githubusercontent.com/398365/40911522-45ff2d6a-67ef-11e8-92a1-704fb3533f0b.png">

I waited for a while, but it seems like nothing was happening.

Eventually I closed Tick Tack and went back to Patchwork to share this feedback.

It was only after reopening Tick Tack to grab some extra screenshots and leaving it running in the background that content actually appeared.

I feel that there should be more indication that there is syncing going on in the background, ideally with some indicator of progress.

Anyways you probably already know about all of this, but I figured the feedback could be helpful.

Please feel free to close this issue whenever 😄

**Comment by ticktackim** [comment]
Thanks for your comments. We will keep polishing it.


原始邮件
发件人:Nick Johnstonenotifications@github.com
收件人:ticktackim/ticktack-workplanticktack-workplan@noreply.github.com
抄送:Subscribedsubscribed@noreply.github.com
发送时间:2018年6月4日(周一) 18:06
主题:[ticktackim/ticktack-workplan] v0.4.0 First impressions (#160)


Hiya!
Congratulations on your v0.4.0 beta release.
Thought I would download it and have a poke around.
Here are some relatively raw first impressions for you:
After opening Tick Tack, I was greeted with the new user page. I was quite nervous about this page after my previous adventure of accidentally forking my feed with Patchwork, so I cancelled out of it.
I was then keen to poke around and see what content in tick tack looks like.
Unfortunately my discover feed is empty!
It's not clear to me if syncing is in progress, or if anything will ever appear here, or if it's because I haven't done something?
I figured then that I should subscribe to some stuff, so I click the Subscriptions button. I see this:
I waited for a while, but it seems like nothing was happening.
Eventually I closed Tick Tack and went back to Patchwork to share this feedback.
It was only after reopening Tick Tack to grab some extra screenshots and leaving it running in the background that content actually appeared.
I feel that there should be more indication that there is syncing going on in the background, ideally with some indicator of progress.
Anyways you probably already know about all of this, but I figured the feedback could be helpful.
Please feel free to close this issue whenever 😄

**Comment by mixmix** [comment]
hey @Widdershin the "sync'ing"  issue is actually an indexing issue. This is being tracked + worked on over here : https://github.com/ticktackim/ticktack-workplan/issues/159

---

> After opening Tick Tack, I was greeted with the new user page. I was quite nervous about this page after my previous adventure of accidentally forking my feed with Patchwork, so I cancelled out of it.

This part I'm more interested in. What did you see what you first opened Ticktack? I'm assuming you were opening it on a computer you had an ssb account on already (by using Patchwork). The paths in which are possible are : 
1. You see a "create new account / import account" page
2. You see a modal showing you your current identity and options to edit it
  - if indexing was happening, your current name / image may not have loaded, making it look like a brand new account - **I think this is the path your were on**

**Comment by Widdershin** [comment]
Yeah I think you're right that I was on 2., couldn't see my image but it had actually loaded my name. Just wasn't sure if it was derived my from system account. In any case the next time I returned to it my image had loaded so I felt like it was okay.
